boxsdk/client/client.py

```diff
@@ -317,6 +318,39 @@ def create_group(self, name):
         )
 
     @api_call
+    def get_recent_items(self, limit=None, marker=None, fields=None, **kwargs):
```

Comment:
Cool, I was going to do this, but noticed it wasn't done that way somewhere else, so I thought it might be like, a standard.